contrib/friendz/src/friendz_android.nit

```diff
@@ -21,11 +22,20 @@ redef class App
 	redef fun window_created
 	do
 		super
-
 		var h = screen_height
 		display.set_viewport(0,0,h*display.width/display.height,h)
 
 		zoom = display.height.to_f / h.to_f
+		default_mediaplayer.looping = true
+	end
+
+	redef fun input(input_event)
+	do
+		if input_event isa KeyEvent then
+			return false
+		else
+			return super
+		end
```

Comment:
This is uselessly hackish. The previous def deal with KeyEvents, but it does return true for all keys. Just update onKeyDown to return true only on intercepted events and it will fix your volume keys problem.